core/store/tx_manager.go

```diff
@@ -685,14 +685,15 @@ func (txm *EthTxManager) handleSafe(
 }
 
 func (txm *EthTxManager) BumpGasByIncrement(originalGasPrice *big.Int) *big.Int {
-	return BumpGas(txm.config, originalGasPrice)
+	gasPrice, _ := BumpGas(txm.config, originalGasPrice)
```

Comment:
Should this error be logged!

Reply:
Its for the legacy tx manager which isnt used any more.

I just havent got around to deleting the code yet.